go.mod

```diff
@@ -1,26 +1,24 @@
 module github.com/gardener/oidc-webhook-authenticator
 
-go 1.15
+go 1.16
 
 require (
-	github.com/coreos/go-oidc v2.1.0+incompatible
+	github.com/coreos/go-oidc/v3 v3.1.0
 	github.com/go-logr/logr v0.4.0
 	github.com/onsi/ginkgo v1.16.4
 	github.com/onsi/gomega v1.13.0
-	github.com/spf13/cobra v1.1.1
+	github.com/spf13/cobra v1.1.3
 	github.com/spf13/pflag v1.0.5
-	golang.org/x/time v0.0.0-20210220033141-f8bda1e9f3ba
-	gopkg.in/square/go-jose.v2 v2.5.1
-	k8s.io/api v0.21.1
-	k8s.io/apimachinery v0.21.1
-	k8s.io/apiserver v0.21.1
-	k8s.io/client-go v0.21.1
-	k8s.io/component-base v0.21.1
-	k8s.io/utils v0.0.0-20210527160623-6fdb442a123b
+	golang.org/x/time v0.0.0-20210723032227-1f47c861a9ac
+	gopkg.in/square/go-jose.v2 v2.6.0
+	k8s.io/api v0.22.2
+	k8s.io/apimachinery v0.22.2
+	k8s.io/apiserver v0.22.2
+	k8s.io/client-go v0.22.2
+	k8s.io/component-base v0.22.2
+	k8s.io/utils v0.0.0-20210819203725-bdf08cb9a70a
 	sigs.k8s.io/controller-runtime v0.9.0
 )
 
 // keep this in sync with k8s.io/apiserver version
-replace github.com/googleapis/gnostic => github.com/googleapis/gnostic v0.4.1 // 0.4.1 is for k8s.io/apiserver@v0.20.2
-
-replace k8s.io/apiserver => k8s.io/apiserver v0.0.0-20210412033426-c5d971fadc40
+replace github.com/googleapis/gnostic => github.com/googleapis/gnostic v0.5.5 // v0.5.5 is for k8s.io/apiserver@v0.22.2
```

Comment:
This can be like 
```suggestion
replace (
	// keep this in sync with k8s.io/apiserver version
	github.com/googleapis/gnostic => github.com/googleapis/gnostic v0.5.5 // v0.5.5 is for k8s.io/apiserver@v0.22.2
)
```

Reply:
Hmm, yes but `go mod tidy` returns it back to a single line so I guess it is OK to leave it like this for the time being.

Reply:
Good catch, I was not aware `tidy` is doing it. Then it is fine.